vehicle/GasEngine.h

```diff
@@ -0,0 +1,22 @@
+#ifndef GAS_ENGINE_H
+#define GAS_ENGINE_H
+
+#include "IEngine.h"
+
+class GasEngine : public Engine {
+public:
+    GasEngine(int cylinders) : cylinders(cylinders) {}
+
+    std::string getFuelType() const override {
+        return "Gas";
+    }
+
+    int getNumberOfCylinders() const override {
+        return cylinders;
+    }
+
+private:
+    int cylinders;
```

Comment:
use some name convention: give some prefix to class member names

Reply:
names changed